src/components/TextInput/TextInput.js

```diff
@@ -613,7 +613,7 @@ class TextInput extends React.Component<Props, State> {
                 <LeftSideWrapper>
                   {(innerImageURI || fallbackSource) && <Image
                     source={imageSource}
-                    fallbackSource={fallbackSource}
+                    fallbackSource={typeof innerImageURI === 'number' ? imageSource : fallbackSource}
```

Comment:
What's with number type check here?

Reply:
so sometimes the picture wouldn't render because it's expecting an URI source, but if the resource is a local require(x) file it will be rendered to fallback